tests/testthat/test-plotly.R

```diff
@@ -358,3 +358,17 @@ test_that("Line breaks are properly translated (R -> HTML)", {
 test_that("group_by() on a plotly object doesn't produce warning", {
    expect_warning(group_by(plot_ly(txhousing), city), NA)
 })
+
+test_that("Check QTables dont cause errors", {
+    .createQTableArray <- function(n, class.name)
+        structure(
+            array(runif(n), dim = c(n, 1L)),
+            class = class.name
+        )
+    s <- data.frame(
```

Comment:
This results in `Error in as.data.frame.default(x[[i]], optional = TRUE) : 
  cannot coerce class ‘"QTable"’ to a data.frame`
for me with R v4.1.2 and v4.2.3

Reply:
Thanks for checking, test at https://app.displayr.com/Dashboard?project_id=-1047844#page=69b42f15-377d-4a9f-930d-33f6308a06f8

Reply:
Still seeing the same error trying to create the `s` data.frame in the test.

Reply:
Should be fixed in b95fd2ffffc70bd31b90520d5074ddc74e11df2d